app/assets/stylesheets/spacex/style.css

```diff
@@ -0,0 +1,4779 @@
+/*
+Version 2.0
+[Table of contents]
+-------------------------------------------------------------------------
+0. Import file styles
+1. Globally Applied Styles
+2. Layout/Boxed
+3. Hover
+4. Support Class
+5. Main Menu
+6. Structure
+7. Header
+8. User Main Menu
+9. Navigation
+10. Fixed Navbar
+11. Footer
+12. Box Content
+13. Calendar
+14. Chartist Chart
+15. Checkbox
+16. Data Tables
+17. Dropcap
+18. Example Content
+19. Inbox List
+20. Jquery UI
+21. Menu Mobile Button
+22. Morris Chart
+23. Notice
+24. noUiSlider
+25. Process Bar
+26. Radio
+27. Search Form
+28. Statistics Box
+29. Switch
+30. Tabs
+31. Timepicke
+32. Title
+33. User Info
+34. Widget Stat
+35. Widget Stat Chart
+36. Misc
+37. Mailbox
+38. Login, Register, ... Form
+39. 404,500 Page
+40. Projects Page
+41. Task Board
+42. Profile
+43. Contact List
+44. Pricing Plan
+45. Invoice
+46. Gallery
+47. Modal
+48. Form Control
+49. Reviews Widget
+50. Activity Widget
+51. Todo Widget
+52. X-Editable
+53. Sparkline Chart
+54. SweetAlert
+-------------------------------------------------------------------------*/
+/*-----------------------------------------------------------------------
+0. Import file styles
+*/
+@import 'https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700';
+@import url(../fonts/font-awesome/css/font-awesome.min.css);
```

Comment:
is this working w/ rails assets pipeline?

Reply:
connect as gem (and check what version it's using and find appropriate gem